Core/Core/Serialisation/BaseObjectSerialzerUtilities.cs

```diff
@@ -217,11 +217,25 @@ internal static Type GetAtomicType(string objFullType)
       var type = KitManager.Types.FirstOrDefault(tp => tp.FullName == typeName);
       if (type != null)
         return type;
+
+      //To allow for backwards compatibility saving deserialization target types.
+      //We also check a ".Deprecated" prefixed namespace
+      string deprecatedTypeName = GetDeprecatedTypeName(typeName);
+
+      var deprecatedType = KitManager.Types.FirstOrDefault(tp => tp.FullName == deprecatedTypeName);
+      if (deprecatedType != null)
+        return deprecatedType;
     }
 
     return typeof(Base);
   }
 
+  internal static string GetDeprecatedTypeName(string typeName, string deprecatedSubstring = "Deprecated.")
```

Comment:
Is it `.Deprecated` or `Deprecated.`? The comment above seems to be conflicting with the default value